### PR TITLE
move yaml lib into artcommon lib

### PR DESCRIPTION
**Pull request by Ximinhan**
Add yaml_handler function to artcommon lib to set yaml config and load by others to reduce duplication
also set indent config for yaml, this will change list item
```
arches:
- x86_64
- ppc64le
- s390x
- aarch64
```
to
```
arches:
  - x86_64
  - ppc64le
  - s390x
  - aarch64
```
for better reading

### CODE REVIEW

File: artcommon/artcommonlib/util.py [hunk 1]
```diff
@@ -34,6 +34,15 @@ def remove_suffix(s: str, suffix: str) -> str:
         return s[:]
 
 
+def yaml_handler():
```
Comment (vfreex) [inline artcommon/artcommonlib/util.py:37]: This function needs more describable name, like `new_roundtrip_yaml_processor`

Comment (vfreex) [inline artcommon/artcommonlib/util.py:37]: This function needs more describable name, like `new_roundtrip_yaml_processor`

Comment (Ximinhan) [reply artcommon/artcommonlib/util.py]: updated

**Review by vfreex — APPROVED** [review]
/lgtm

**Pull request merged**